File: 2015/08/2.rb
chars = 0

File.open('input.txt').each do |line|
  line.chomp!
  len = line.length
  line.gsub!(/^"|"$/, '.' * 3)
  line.gsub!(/\\\\|\\"/, '.' * 4)
  line.gsub!(/\\x../, '.' * 5)
  chars += line.length - len
end

puts chars

File: 2024/01/2.py
from collections import defaultdict

list1 = []
list2 = defaultdict(int)

with open("input.txt") as f:
    for line in f:
        int1, int2 = map(int, line.split())
        list1.append(int1)
        list2[int2] += 1

answer = sum(i * list2[i] for i in list1)

print(answer)
